app/views/plantings/index.html.haml

```diff
@@ -8,11 +8,15 @@
       %p
         - if @owner == current_member
           = link_to 'Plant something', new_planting_path, :class => 'btn btn-primary'
-        = link_to "View everyone's plantings", plantings_path, :class => 'btn btn-default'
+        - content_for :buttonbar do
+          = link_to 'My Active Plantings', plantings_by_owner_path(:owner=>current_member.slug), :class => 'btn btn-default'
+          = link_to 'All My Plantings', plantings_by_owner_path(:all=>1, :owner=>current_member.slug), :class => 'btn btn-default'
+          = link_to "Everyone's plantings", plantings_path, :class => 'btn btn-default'
     - else # everyone's plantings
       = link_to 'Plant something', new_planting_path, :class => 'btn btn-primary'
       - if current_member
         = link_to 'View your plantings', plantings_by_owner_path(:owner => current_member.slug), :class => 'btn btn-default'
+      = link_to 'Show all (including inactive)', plantings_path(:all=>1)
```

Comment:
Sure. Right now, the "Everyone's plantings, including inactive" view looks like this:
![screenshot from 2017-01-05 12-12-08](https://cloud.githubusercontent.com/assets/225991/21680208/44ca89a2-d340-11e6-8b02-78ffd6e94dd9.png)

The "Show all (including inactive)" link takes you to the page you're already looking at, so it isn't useful. AFAICT, if you want to see only active plantings, you need to manually edit the URL bar, or perhaps hit "Back" if that's where you came from. I'm suggesting making that link into a toggle rather than a one-way journey.